packages/protocol/contracts/tokenvault/ERC20Vault.sol

```diff
@@ -87,6 +88,21 @@ contract ERC20Vault is BaseVault {
     error VAULT_MESSAGE_NOT_FAILED();
     error VAULT_MESSAGE_RELEASED_ALREADY();
 
+    /// @dev If a native token issuer (e.g.: Circle/Lido) revokes minter role from our contracts
+    /// (once they get the ownership), we won't be able to mint / burn the tokens. But we still want
+    /// to mint these tokens like "USDC ⭀31337" style, but then we need to reset the old mapping.
```

Comment:
Last part  about `"USDC ⭀31337" style` could use some more information because I don't really understand what it means from reading this.

Reply:
the name and symbol has been changed in another PR. This ⭀ is removed.

Reply:
> Last part about `"USDC ⭀31337" style` could use some more information because I don't really understand what it means from reading this.

It was just an example. (And we used these namings for `BridgedERC20` assets)
So the situation (I tried to explain) as follows:
1. Taiko predeploys USDC on L2.
2. People bridging USDC into that contract (via the translator obv.)
3. The ownership of USDC on L2 will be transferred eventually to Circle.
4. Circle for whatever reason decides to cut the translator/tokenvault's roles (minter role).

In this case we need to reset the mapping too in the registry, but also has to reset this `canonicalToBridged[chainId][canonicalAddrToReset]`, otherwise we will not be able to bridge USDC anymore (like` USDC ⭀31337` as our `BridgedERC20` token, hence the mapping points to the contract for whcih we has no minter rights anymore.)